content/04.experiment-types.md

```diff
@@ -37,21 +37,20 @@ Additionally, phosphoproteomic analysis is often time-consuming and requires the
 
 #### Glycoproteomics
 One PTM gaining interest due to its ubiquity and emerging functional roles is glycosylation. 
-Protein glycosylation sites can be N-linked (asparagine-linked), O-linked (serine/threonine-linked). 
+Protein glycosylation sites can be N-linked (asparagine-linked) or O-linked (serine/threonine-linked). 
 Understanding the function of protein glycosylation will help us understand numerous biological processes since this is a universal protein modification across all domains of life, especially at the cell surface [@doi:10.1038/s43586-022-00128-4; @doi:10.1093/glycob/cww086; @doi:10.1016/j.cell.2006.08.019; @doi:10.1146/annurev-neuro-071714-034019].
 
 Studies of phosphorylation and glycosylation share several experimental pipeline steps including sample preparation. 
 Protein clean-up approaches for glycoproteomics may differ from other proteomics experiments because glycopeptides are more hydrophilic than most peptides. 
 Some approaches mentioned in literature include: filter-aided sample preparation (FASP), suspension traps (S-traps), and protein aggregation capture (PAC) [@doi:10.1038/s43586-022-00128-4; @doi:10.1038/nmeth.1322 ; @doi:10.1002/pmic.201300553; @doi:10.1074/mcp.TIR118.001270; @doi:10.1021/acs.jproteome.8b00505; @doi:10.1021/acs.analchem.0c04726; @doi:10.26434/chemrxiv.13487364.v1].
 Multiple proteases may be used to increase the sequence coverage and detect more modification sites, such as: trypsin, chymotrypsin, Pepsin, WaLP/MaLP [@DOI:10.1074/mcp.M113.034710], GluC, AspN, Pronase, Proteinase K,  OgpA, StcEz, BT4244, AM0627, AM1514, AM0608, Pic, ZmpC, CpaA, IMPa, PNGase F, Endo F, Endo H, and OglyZOR [@doi:10.1038/s43586-022-00128-4].
-Mass spectrometry has improved over the past decade so that now more strategies are available for glycoprotein structure elucidation and also for glycosylation site quantification [@doi:10.1038/s43586-022-00128-4].
-
+Mass spectrometry has improved over the past decade so that now more strategies are available for glycoprotein structure elucidation and also for glycosylation site quantification [@doi:10.1038/s43586-022-00128-4]. See also section "AminooxyTMT Isobaric Mass Tags" for more on quantitative glycoproteomics.
```

Comment:
```suggestion
Mass spectrometry has improved over the past decade so that now more strategies are available for glycoprotein structure elucidation and also for glycosylation site quantification [@doi:10.1038/s43586-022-00128-4]. 
See also section "AminoxyTMT Isobaric Mass Tags" for more on quantitative glycoproteomics.
```